tests/dummy/app/templates/map.hbs

```diff
@@ -20,6 +20,22 @@
           <i class="info circle icon"></i>
         </button>
       </div>
+      <div class='ui compact menu' style='border-width:0px;border-image-width:0px;box-shadow: rgba(0, 0, 0, 0) 0px 0px 0px 1px inset; hight:36px;'>
```

Comment:
I think we should wrap every maptool into component, but it is separate task
But for a while, move styles into CSS, please, into ./addon/styles/maptools/measure.scss and import this style inside ./addon/styles/addon.scss.